docs/operate-and-deploy/installation/server-config/config-reference.md

```diff
@@ -612,11 +612,13 @@ property has the value `KSQL_PROCESSING_LOG`.
 Toggles whether or not the processing log should include rows in log
 messages. By default, this property has the value `false`.
 
-### ksql.logging.server.skipped.response.codes
+### ksql.logging.server.rate.limited.response.codes
 
-A comma-separated list of HTTP response codes to skip during server
-request logging. This is useful for ignoring certain 4XX errors that you
-might not want to show up in the logs.
+A list of `path:rate_limit` pairs, to limit the rate of server request
+logging. This is useful for limiting certain 4XX errors that you
+might not want to blow up in the logs. 
+This setting enables seeing the logs when the request rate is low 
+and dropping them when they go over the threshold.
 
 ### ksql.logging.server.rate.limited.request.paths
```

Comment:
We're currently logging for internal endpoints (used in server-to-server communication for multi-node clusters), right? Would it make sense to disable logging for those by default? (I don't feel strongly one way or the other -- you have more context than I do about when/how often those endpoints are hit.)